core/src/test/java/org/neo4j/ogm/domain/friendships/Person.java

```diff
@@ -26,7 +26,7 @@
     private Long id;
     private String name;
 
-    @Relationship(type = "FRIEND_OF")
+    @Relationship(type = "FRIEND_OF", direction = Relationship.UNDIRECTED)
```

Comment:
The default direction is outgoing. That might have been the purpose- is it failing with outgoing now?